src/main/java/com/qaprosoft/carina/demo/gui/pages/ModelInfoPage.java

```diff
@@ -37,9 +39,14 @@
     public ModelInfoPage(WebDriver driver) {
         super(driver);
     }
+    
+   
 
     public String readDisplay() {
         assertElementPresent(displayInfoLabel);
+         if (s2.isEmpty()) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') A "NullPointerException" could be thrown; "s2" is nullable here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://demo.qaprosoft.com/sonarqube/coding_rules#rule_key=squid%3AS2259)